easybuild/cray_external_modules_metadata.cfg

```diff
@@ -1,5 +1,15 @@
 # metadata useful to EasyBuild (hpcugent.github.io/easybuild) for modules provided by Cray on Cray systems
-# authors: Guilherme Peretti-Pezzi (CSCS), Petar Forai (IMP/IMBA), Kenneth Hoste (HPC-UGent)
+# authors: Guilherme Peretti-Pezzi (CSCS), Petar Forai (IMP/IMBA), Kenneth Hoste (HPC-UGent), Eirini Koutsaniti (CSCS)
+
+[cray-R/3.3.3]
+name = CrayR
+version = 3.3.3
+prefix = /opt/R/3.3.3
+
+[cray-R/3.4.2]
+name = CrayR
+version = 3.4.2
+prefix = /opt/R/3.4.2
```

Comment:
Looks like you've ordered the entries by taking the case into account, as normally `sort` would do. This can be a bit confusing for human beings for mixed case inputs. I would suggest to move `cray-R` entries down where they were placed initially, as if you were ignoring the case.